framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1781,6 +1781,23 @@ public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree lhsTree) {
         return res;
     }
 
+    /**
+     * As long as everUseFlow is true, always enable flow refinement for the receiver. This method
+     * is an implementation detail and visible inside the package only. See the comment in this
+     * testcase for more details framework/tests/viewpointtest/TestgetAnnotatedLHS.java.
```

Comment:
Please rename the test to `TestGetAnnotatedTypeLhs`. The `LHS` should be capitalized consistently with the method name. I would upper-case the `G` to make the split between words clearer.